nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const fs = require("fs");
+
+const app = express();
+
+const meals = JSON.parse(fs.readFileSync(__dirname + "/data/meals.json"));
+const reviews = JSON.parse(fs.readFileSync(__dirname + "/data/reviews.json"));
+const reservations = JSON.parse(fs.readFileSync(__dirname + "/data/reservations.json"));
```

Comment:
Great Work!!! I like the use of one variable to parse and sync the file. 

I realised only now after looking at your homework, that I have not worked on updated homework.I was assuming that only 'fs' was added to the new one and just continued with old.